app/assets/stylesheets/partials/_layouts.scss

```diff
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+}
+
+body {
+  background-color: white;
+}
+
+.content-container {
+  padding-left: $nav-left-width + 30px;
```

Comment:
not sure but this might cause an error since you don't have `$nav-left-width` defined in `variables.scss`